prometheus_aioexporter/script.py

```diff
@@ -134,6 +139,24 @@ def get_parser(self) -> argparse.ArgumentParser:
             action="store_true",
             help="include process stats in metrics",
         )
+        parser.add_argument(
+            "--ssl-private-key",
+            type=str,
+            dest="ssl_private_key",
+            help="full path to the ssl private key",
+        )
+        parser.add_argument(
+            "--ssl-public-key",
+            type=str,
+            dest="ssl_public_key",
+            help="full path to the ssl public key",
+        )
+        parser.add_argument(
+            "--ssl-ca",
+            type=str,
+            dest="ssl_ca",
+            help="full path to the ssl certificate authority (CA)",
+        )
```

Comment:
Thanks so much. I am using the `argparse.FileType()` now. However, unistests started failing locally now. `trustme` generates certificates in a temporary folder. The `load_verify_locations` of the `ssl` module fails because of that temporary file. I will continue testing.

Reply:
I found and fixed the issue! 
The `load_verify_locations` of the `ssl` module expects the path to the certificate components as `str`. I was providing it with `FileType` objects and this is why it was failing. It was related with the temporary file and `trustme`. I am resolving this converasation